web/sharings/sharings_test.go

```diff
@@ -187,21 +210,29 @@ func TestCreateSharingSuccess(t *testing.T) {
 func TestMain(m *testing.M) {
 	config.UseTestFile()
 	testutils.NeedCouchdb()
-	setup := testutils.NewSetup(m, "sharing_test")
-	testInstance = setup.GetTestInstance()
-	instanceURL, _ = url.Parse("https://" + testInstance.Domain + "/")
+
+	setup := testutils.NewSetup(m, "sharing_test_alice")
+	setup2 := testutils.NewSetup(m, "sharing_test_bob")
+	testInstance = setup.GetTestInstance(&instance.Options{
+		PublicName: "Alice",
+	})
+	_ = setup2.GetTestInstance(&instance.Options{
+		PublicName: "Bob",
+	})
 
 	jar = setup.GetCookieJar()
 	client = &http.Client{
 		CheckRedirect: noRedirect,
 		Jar:           jar,
 	}
-
 	clientOAuth, _ = setup.GetTestClient("")
 	clientID = clientOAuth.ClientID
 
 	ts = setup.GetTestServer("/sharings", Routes)
+	ts2 = setup2.GetTestServer("/auth", auth.Routes)
+
 	os.Exit(setup.Run())
+	os.Exit(setup2.Run())
```

Comment:
setup2 will never run, (os.Exit closes the process), so the second Instance will not be cleaned up.
It is a bit ugly, but you can add a public Cleanup to test_utils
```
func (c *TestSetup) Cleanup() {
	c.cleanup()
}
```

and then link the two testSetup
```
setup.AddCleanup(setup2.Cleanup)
```